setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-__version__ = "0.2.0"
+__version__ = "0.2.1"
```

Comment:
This is a new feature, not a big fix, so it should be a minor version change, not a patch version.